mlrun/datastore/utils.py

```diff
@@ -16,7 +16,11 @@
 
 
 def store_path_to_spark(path):
-    if path.startswith("v3io:///"):
+    if path.startswith("redis://") or path.startswith("rediss://"):
+        url = urlparse(path)
+        if url.path:
+            return url.path
```

Comment:
suggest keeping with the function structure of setting "path" and returning it in the end of function.
also what is the handling if not url.path exist?

Reply:
Will move. Non existing url.path is handled by the caller